examples/express/with-emailpassword/frontend/package.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "supertokens-react",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "@testing-library/jest-dom": "^5.16.5",
+        "@testing-library/react": "^13.4.0",
+        "@testing-library/user-event": "^13.5.0",
+        "@types/jest": "^27.5.2",
+        "@types/node": "^16.11.56",
+        "@types/react": "^18.0.18",
+        "@types/react-dom": "^18.0.6",
+        "axios": "^0.21.0",
+        "react": "^18.2.0",
+        "react-dom": "^18.2.0",
+        "react-router-dom": "^6.2.1",
+        "react-scripts": "5.0.1",
+        "supertokens-auth-react": "latest",
+        "typescript": "^4.8.2",
+        "web-vitals": "^2.1.4"
```

Comment:
this should be empty cause all these should be in the package.json of the with-emailpassword folder.